### DISCUSSION TITLE
I would like to support the project

### DISCUSSION THREAD

**Discussion by ounokoya** [coding-kitties/investing-algorithm-framework#272] [Q&A] [answered]

**Comment by MDUYN** [comment, marked as answer]
Hey @ounokoya, thanks for reaching out. There is a contributing guideline that can be found [here](https://github.com/coding-kitties/investing-algorithm-framework/blob/main/docs/CONTRIBUTING.md). If you would like to pick up a task that is part of the overall development please pickup a task from the project board that can be found [here](https://github.com/orgs/coding-kitties/projects/3). If you need any help with the tasks, feel free to start a discussion [here](https://github.com/coding-kitties/investing-algorithm-framework/discussions).
